docs/source/serialization.rst

```diff
@@ -16,23 +16,10 @@ Here is the serialization and deserialization flow::
       quantize_,
       Int4WeightOnlyConfig,
   )
-
-  class ToyLinearModel(torch.nn.Module):
-      def __init__(self, m=64, n=32, k=64):
-          super().__init__()
-          self.linear1 = torch.nn.Linear(m, n, bias=False)
-          self.linear2 = torch.nn.Linear(n, k, bias=False)
-
-      def example_inputs(self, batch_size=1, dtype=torch.float32, device="cpu"):
-          return (torch.randn(batch_size, self.linear1.in_features, dtype=dtype, device=device),)
-
-      def forward(self, x):
-          x = self.linear1(x)
-          x = self.linear2(x)
-          return x
+  from torchao.testing.model_architectures import ToyTwoLinearModel
```

Comment:
can you revert the changes for this? I think it's better to have this tutorial self contained